### ISSUE TITLE
Add circular seconds

### ISSUE THREAD

**Issue by spikyjt** [spikyjt/SailingTimer#2]
Add an arc around the watch face perimeter to track seconds.

This should flash during notifications, particularly at start.

**Comment by duckpaint** [comment]
Yes! it would be great that the arc in the watch face perimeter is the same length of  all the minutes in the countdown!
